site/examples/navigation.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const menuNavigation = `<nav'>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3 is-active' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-home'/></svg> Home
+  </a>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-user'/></svg> Account
+  </a>
+  <a class='btn color-blue color-white-on-active bg-transparent bg-darken5-on-hover bg-blue-on-active txt-s ml3' href='#'>
+    <svg class='icon inline-block align-middle'><use xlink:href='#icon-logout'/></svg> Logout
+  </a>
+</nav>`;
+
+const underlinedTabs = `<div class='bg-gray border-b border--black flex-parent flex-parent--stretch-cross h40 txt-bold txt-s'>
```

Comment:
Because the body background is so dark, it's hard to see what's going on here — that the white border is overlapping a black border.